npc/re/merchants/shops.txt

```diff
@@ -91,6 +91,25 @@ malangdo,232,163,5	shop	Tool Dealer Myo	560,611:-1,1750:-1,645:-1,656:-1,601:-1,
 mora,100,118,4	shop	Tool Dealer#mora	517,11522:-1,11523:-1,11524:-1,602:-1,601:-1,611:-1,1773:-1,1774:-1
 mora,138,110,4	shop	Mora Chef	518,11519:-1
 
+//=======================================================
+// Morroc
+//=======================================================
+morocc,156,76,4	shop	Butcher#moc1	58,517:-1
+moc_ruins,91,128,4	shop	Tool Dealer#moc1	93,1750:-1,611:-1,501:-1,502:-1,503:-1,504:-1,506:-1,645:-1,656:-1,601:-1,602:-1,1065:-1
```

Comment:
This one is pre-re only.

Reply:
according kRO navi_npc_krpri.lub, it's correct

Reply:
Weird. Hmmm ok i'll check this and the other "errors" ingame on iRO and aegis 15.2 as soon as I can.

Reply:
![imagen](https://user-images.githubusercontent.com/4557962/37288577-ed156a4e-2607-11e8-8933-ffe03349830a.png)
Yep, it's there, but moved 1 tile to the left for some reason (?), coordinates on navi_npc_krpri.lub and my aegis file are the same as yours tho (wtf iRO D:)

Reply:
It should be on the main file then, because it's present in both pre-re and re.

Reply:
hmm, no
there are item differences between these 2, that I forgot to update.